src/test/java/ch/uzh/ifi/hase/soprafs22/controller/UserControllerTest.java

```diff
@@ -35,7 +35,7 @@
  * This tests if the UserController works.
  */
 @WebMvcTest(UserController.class)
-public class UserControllerTest {
```

Comment:
Why did you change it from public to protected?

Reply:
a sonarQube thing;)